doc/working_example.md

```diff
@@ -0,0 +1,824 @@
+# Genalyzer Through Examples
+## A Minimal Working Example
+The workflow of Genalyzer is best understood with the help of a working example. We use Python to illustrate this workflow in the current section. The data that we will process using Genalyzer for this example is also generated using Genalyzer. It consists of a ``300 KHz`` tone sampled at ``3 MSPS``. The only source of noise in this synthetic data is quantization noise.
+<br/><br/>
+
+### Compute FFT
+We begin with the initial steps involved, _i.e.,_, configuring Genalyzer and performing FFT on the generated data. Since spectral analysis is the core of Genalyzer, computing FFT is the first important step. Expand the code-snippet below for details.
+<details>
+  <summary>Code Snippet</summary>
+
+``` Python
+def main():
```

Comment:
Remove use of the main function abstraction. It just makes the examples confusing and wastes space

Reply:
Updated.